libs/langgraph/tests/test_pregel.py

```diff
@@ -2625,6 +2625,9 @@ class StateUpdate(BaseModel):
         answer: Optional[str] = None
         docs: Optional[list[str]] = None
 
+    class UpdateDocs34(BaseModel):
+        docs: list[str] = ["doc3", "doc4"]
```

Comment:
Should we add another field and set it to "None" explicitly to avoid regressions on that or is that captured well by snapshot tests?